cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -395,3 +423,31 @@ def _systemctl_is_active(application):
         return b'active' in raw
     except Exception:
         return False
+
+
+def _apply_node_label(label, delete=False):
+    ''' Invoke kubectl to apply node label changes '''
+
+    hostname = gethostname()
+    # TODO: Make this part of the kubectl calls instead of a special string
+    cmd_base = 'kubectl --kubeconfig=/srv/kubernetes/config label node {0} {1}'
+
+    if delete is True:
+        label_key = label.split('=')[0]
+        cmd = cmd_base.format(hostname, label_key)
+        cmd = cmd + '-'
```

Comment:
I don't actually know what this does `kubectl label node juju-xdsf1 foo=bar-` ?  Why the trailing hyphen?